Content.Server/Atmos/Piping/Unary/EntitySystems/GasVentPumpSystem.cs

```diff
@@ -112,7 +112,7 @@ private void OnGasVentPumpUpdated(EntityUid uid, GasVentPumpComponent vent, ref
                 {
                     // Leak only a small amount of gas as a proportion of supply pipe pressure.
                     var pipeDelta = pipe.Air.Pressure - environment.Pressure;
-                    transferMoles = (float)timeDelta * pipeDelta * vent.UnderPressureLockoutLeaking;
+                    transferMoles = (float) timeDelta * pipeDelta * vent.UnderPressureLockoutLeaking;
```

Comment:
Extraneous space

Reply:
I only changed that because VSCodium was complaining. I saw the discussion on github, is the .editorconfig changed by now? I can remove the space, no problem.

Reply:
Looks like this is the only nitpick we're waiting on. Please also try to condense it down to two changelog entries if you can.

Reply:
Both are done.